### PR TITLE
Two small fixes to issues discovered by static analysis

### PR DESCRIPTION
**Pull request by msekletar**

### CODE REVIEW

**Comment by msekletar** [comment]
@lathiat it would be awesome if you could review this. Should be very quick. Thanks!

File: avahi-daemon/chroot.c [hunk 1]
```diff
@@ -188,8 +188,8 @@ static int recv_fd(int fd) {
             return -1;
         }
 
-        assert(h->cmsg_len = CMSG_LEN(sizeof(int)));
-        assert(h->cmsg_level = SOL_SOCKET);
+        assert(h->cmsg_len == CMSG_LEN(sizeof(int)));
+        assert(h->cmsg_level == SOL_SOCKET);
```
Comment (pemensik) [inline avahi-daemon/chroot.c:192]: This part has been already proposed in other PR. Have to find its ID.

File: avahi-client/browser.c [hunk 2]
```diff
@@ -72,6 +72,8 @@ static void parse_domain_file(AvahiDomainBrowser *b) {
         if (avahi_normalize_name(buf, domain, sizeof(domain)))
             b->static_browse_domains = avahi_string_list_add(b->static_browse_domains, domain);
     }
+
+    fclose(f);
```
Comment (pemensik) [inline avahi-client/browser.c:76]: Thanks, definitely needed here!

**Review by pemensik — APPROVED** [review]

File: avahi-daemon/chroot.c (hunk 1 above)
Comment (pemensik) [reply avahi-daemon/chroot.c:192]: This is also fixed in PR #372 .

Comment (msekletar) [reply avahi-daemon/chroot.c:192]: So let's close #372 as a duplicate of this one?

**Review by pemensik — APPROVED** [review]

**Comment by evverx** [comment]
I think Coverity is right here but to be honest I'm not sure why the assertions are used there in the first place. They can be compiled out when `NDEBUG` is defined and looking at https://github.com/lathiat/avahi/pull/17 it appears `NDEBUG` is actually used to avoid a bunch of debug messages.

**Pull request merged**